tasks/eth/027-holocene-system-config-upgrade-and-init-multi-chain/.env

```diff
@@ -0,0 +1,8 @@
+ETH_RPC_URL="https://ethereum.publicnode.com"
+OWNER_SAFE=0x5a0Aae59D09fccBdDb6C6CcEB07B7279367C3d2A
+COUNCIL_SAFE=0xc2819DC788505Aac350142A7A707BF9D03E3Bd03
+FOUNDATION_SAFE=0x847B5c174615B1B7fDF770882256e2D3E95b9D92
+
+SAFE_NONCE_0x5A0AAE59D09FCCBdDb6C6CCEB07B7279367C3d2A=8
```

Comment:
we should make sure the full address is in uppercase here.

Reply:
[363b7ba](https://github.com/ethereum-optimism/superchain-ops/pull/454/commits/363b7ba2590fd33988e0fc570bebedebc8dc6847)